spec.html

```diff
@@ -11945,7 +11945,7 @@ <h1>Objectives</h1>
 
       <p>This specification does not make any guarantees that any object will be garbage collected. Objects which are not live may be released after long periods of time, or never at all. For this reason, this specification uses the term "may" when describing behaviour triggered by garbage collection.</p>
 
-      <p>The semantics of WeakRef and FinalizationRegistry objects is based on two operations which happen at particular points in time:</p>
+      <p>The semantics of WeakRefs and FinalizationRegistrys is based on two operations which happen at particular points in time:</p>
```

Comment:
oof, `FinalizationRegistrys`

Reply:
Ehhh I will just fix that.

Reply:
lol that's also awkward if FInalizationRegistry is a dfn tho - making this "FinalizationRegistry objects" to avoid this problem was an explicit editorial review item for the weakref proposal :-p

Reply:
Neither `FinalizationRegistrys` nor `FinalizationRegistries` will auto-link currently, so we might as well go with the better one.

Reply:
```suggestion
      <p>The semantics of a WeakRef and a FinalizationRegistry are based on two operations which happen at particular points in time:</p>
```
?

Reply:
That reads worse to me. It's the semantics of _all_ WeakRefs.

To be clear, you're just concerned about the automatic linking, right? I guess this can be the impetus for me to finally fix https://github.com/tc39/ecmarkup/issues/157.

Reply:
I'm concerned about both automatic linking and also natural pluralization; i suppose if ecmarkup could detect `y → ies` then it's not a big deal, but it makes it awkward to speak about out loud.

Reply:
> i suppose if ecmarkup could detect `y → ies` then it's not a big deal

I was thinking I'd just allow `dfn` to specify pluralizations explicitly.

> it makes it awkward to speak about out

Sorry, what becomes awkward?

Reply:
`ys` obviously, when spoken aloud, nobody will come up with the spelling; the `ies` one just seems weird to me even though also obviously, the pluralization of `registry` is `registries`, it's weird when it's a camelcased thing.

not a strong opinion, it's just weird :-)

Reply:
I think FinalizationRegistrys is fine.

Reply:
Rebased on https://github.com/tc39/ecma262/pull/2492, so now the plural `FinalizationRegistrys ` should link.